change/@fluentui-react-breadcrumb-027d665b-7783-4af2-86c0-5cbf7eaa5ec4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30551&commitId=5d130387f6e44a4f2aec5ef695c1651843b221f5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.appearance - Dark Mode.default.chromium.png|36| Changed|
|Breadcrumb Converged.appearance - High Contrast.default.chromium.png|43| Changed|
|Breadcrumb Converged.appearance - High Contrast.pressed.chromium.png|43| Changed|
|Breadcrumb Converged.appearance - RTL.default.chromium.png|37| Changed|
|Breadcrumb Converged.appearance - High Contrast.hover.chromium.png|43| Changed|
|Breadcrumb Converged.appearance - Dark Mode.pressed.chromium.png|36| Changed|
|Breadcrumb Converged.appearance - Dark Mode.hover.chromium.png|36| Changed|
|Breadcrumb Converged.appearance - High Contrast.focused.chromium.png|43| Changed|
|Breadcrumb Converged.appearance - Dark Mode.focused.chromium.png|36| Changed|
|Breadcrumb Converged.appearance - RTL.pressed.chromium.png|37| Changed|
|Breadcrumb Converged.appearance.default.chromium.png|37| Changed|
|Breadcrumb Converged.appearance - RTL.hover.chromium.png|37| Changed|
|Breadcrumb Converged.appearance - RTL.focused.chromium.png|37| Changed|
|Breadcrumb Converged.appearance.focused.chromium.png|37| Changed|
|Breadcrumb Converged.appearance.pressed.chromium.png|37| Changed|
|Breadcrumb Converged.size.hover.chromium.png|85| Changed|
|Breadcrumb Converged.size.default.chromium.png|85| Changed|
|Breadcrumb Converged.size.focused.chromium.png|85| Changed|
|Breadcrumb Converged.appearance.hover.chromium.png|37| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|85| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>